WinImageBuilder.psm1

```diff
@@ -361,14 +361,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]

```

Comment:
commit message: cloudbase-init has two installers -- stable and nightly build (Beta). Use the -BetaRelease flag consistently throughout the patch